src/main/resources/application.yaml

```diff
@@ -8,7 +8,7 @@ zeebe:
       dmn.repository: dmn-repo
 
     job.timeout: 10000
-    broker.contactPoint: 127.0.0.1:26500
+    broker.gateway-address: 127.0.0.1:26500
```

Comment:
Hello @Lzgabel, shouldn't we keep `contactPoint` here? The maven build seems to fail because it cannot reach the Zeebe gateway. WDYT?

Reply:
Hi @vincentgiraud , I've rebased master, try again?

Reply:
Thanks all good